contributing.md

```diff
@@ -1,3 +1,22 @@
 # Contributing
 
-TODO
+First, get the sources of the backend:
+
+``` bash
+curl -L https://github.com/filebrowser/filebrowser/archive/master.tar.gz | tar xvz
```

Comment:
As explained in [this](https://github.com/filebrowser/filebrowser/pull/399#issuecomment-382898612) comment, the objective is to require only docker and a shell. So, assuming that curl is available in almost any shell, the idea is not to require git.

However, I agree with you. Cloning it will encourage peoplo to save changes, at least. I will put both options.